packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/java/integrationsteps.md

```diff
@@ -1,3 +1,72 @@
 ## Integration steps
 
-The Java SDK integration steps for this use case are coming soon. See the [Okta Identity Engine Java SDK usage guide](https://github.com/okta/okta-idx-java#usage-guide) for more information.
+### Step 1: User goes to sign-in page
+
+Build a sign-in page for your app that captures both the username and password.
```

Comment:
For consistency, use "external application" when describing the "thing" that the developer is building outside of Okta and "app integration" when describing the integration that they build using Okta (e.g. using the App Integration Wizard)
I think here you should change this to "Build a sign-in page for your external application that captures the username and password."

Reply:
I think adding "external application" would confuse the user since throughout the guide, this term is not used. It is assumed that the developer is building an app using the OIE Java SDK for embedded auth flow. They will still need to use the App Integration Wizard to register their developed app to Okta.